### DISCUSSION TITLE
Select component doesn't work in windows app

### DISCUSSION THREAD

**Discussion by bahri-dev** [GeekyAnts/NativeBase#5384] [Feature request]
### Description

Select component doesn't work in windows app.

### CodeSandbox/Snack link

https://codesandbox.io/s/frosty-hugle-ex5hgj?file=/src/App.tsx

### Steps to reproduce

2. Click on '...' 
3. Scroll down to '...' 

### NativeBase Version

3.4.15

### Platform

- [ ] Android
- [ ] CRA
- [ ] Expo
- [ ] iOS
- [ ] Next

### Other Platform

window

### Additional Information

Warning: Failed prop type: Invalid props.style key `outlineColor` supplied to `ForwardRef(TextInput)`.
Bad object: {
  "width": "100%",
  "color": "#212121",
  "fontSize": 16,
  "lineHeight": 20,
  "outlineColor": "veneerLight.colorDisabledForeground",
  "paddingRight": 12,
  "paddingLeft": 12,
  "paddingTop": 8,
  "paddingBottom": 8,
  "fontFamily": "Assets/FormaDJRMicro-Regular.ttf#Forma DJR Micro",
  "fontWeight": "400",
  "fontStyle": "normal",
  "backgroundColor": "transparent",
  "flex": 1,
  "height": "100%"
}
Valid keys: [
  "display",
  "width",
  "height",
  "start",
  "end",
  "top",
  "left",
  "right",
  "bottom",
  "minWidth",
  "maxWidth",
  "minHeight",
  "maxHeight",
  "margin",
  "marginVertical",
  "marginHorizontal",
  "marginTop",
  "marginBottom",
  "marginLeft",
  "marginRight",
  "marginStart",
  "marginEnd",
  "padding",
  "paddingVertical",
  "paddingHorizontal",
  "paddingTop",
  "paddingBottom",
  "paddingLeft",
  "paddingRight",
  "paddingStart",
  "paddingEnd",
  "borderWidth",
  "borderTopWidth",
  "borderStartWidth",
  "borderEndWidth",
  "borderRightWidth",
  "borderBottomWidth",
  "borderLeftWidth",
  "position",
  "flexDirection",
  "flexWrap",
  "justifyContent",
  "alignItems",
  "alignSelf",
  "alignContent",
  "overflow",
  "flex",
  "flexGrow",
  "flexShrink",
  "flexBasis",
  "aspectRatio",
  "zIndex",
  "direction",
  "shadowColor",
  "shadowOffset",
  "shadowOpacity",
  "shadowRadius",
  "transform",
  "transformMatrix",
  "decomposedMatrix",
  "scaleX",
  "scaleY",
  "rotation",
  "translateX",
  "translateY",
  "backfaceVisibility",
  "backgroundColor",
  "borderColor",
  "borderTopColor",
  "borderRightColor",
  "borderBottomColor",
  "borderLeftColor",
  "borderStartColor",
  "borderEndColor",
  "borderRadius",
  "borderTopLeftRadius",
  "borderTopRightRadius",
  "borderTopStartRadius",
  "borderTopEndRadius",
  "borderBottomLeftRadius",
  "borderBottomRightRadius",
  "borderBottomStartRadius",
  "borderBottomEndRadius",
  "borderStyle",
  "opacity",
  "elevation",
  "color",
  "fontFamily",
  "fontSize",
  "fontStyle",
  "fontWeight",
  "fontVariant",
  "textShadowOffset",
  "textShadowRadius",
  "textShadowColor",
  "letterSpacing",
  "lineHeight",
  "textAlign",
  "textAlignVertical",
  "includeFontPadding",
  "textDecorationLine",
  "textDecorationStyle",
  "textDecorationColor",
  "textTransform",
  "writingDirection"
]
    at TextInput (http://localhost:8081/index.bundle?platform=windows&dev=true&hot=false&inlineSourceMap=true:83434:39)
    in Unknown
    in RCTView (at View.windows.js:110)
    in View
    in Unknown
    in Box
    in Stack
    in Input
    in RCTView (at View.windows.js:110)
    in View
    in Unknown
    in Box
    in Select
    in $704cf1d3b684cc5c$export$9f8ac96af4b1b2ae (created by NativeBaseProvider)
    in ToastProvider (created by NativeBaseProvider)
    in PortalProvider (created by NativeBaseProvider)
    in HybridProvider (created by NativeBaseProvider)
    in ResponsiveQueryProvider (created by NativeBaseProvider)
    in RCTView (at View.windows.js:110)
    in View (created by CompatNativeSafeAreaProvider)
    in CompatNativeSafeAreaProvider (created by SafeAreaProvider)
    in SafeAreaProvider (created by NativeBaseProvider)
    in NativeBaseConfigProviderProvider (created by NativeBaseProvider)
    in NativeBaseProvider (at App.tsx:58)
    in AppModulesUpdateCheck (at App.tsx:54)
    in Initialization (at App.tsx:46)
    in ErrorBoundary (at App.tsx:45)
    in App (at renderApplication.js:50)
    in RCTView (at View.windows.js:110)
    in View (at AppContainer.js:92)
    in ChildrenWrapper (at wrapRootComponent.js:9)
    in _default (at wrapRootComponent.js:8)
    in Root (at AppContainer.js:107)
    in RCTView (at View.windows.js:110)
    in View (at AppContainer.js:119)
    in AppContainer (at renderApplication.js:43)
    in HP.HPX(RootComponent) (at renderApplication.js:60)
reactConsoleErrorHandler @ :8081/index.bundle?p…ourceMap=true:21601
​ 
        
       Invariant Violation: View config getter callback for component `option` must be a function (received `undefined`). Make sure to start component names with a capital letter.

This error is located at:
    in option
    in select
    in RCTView (at View.windows.js:110)
    in View
    in Unknown
    in Box
    in Select
    in $704cf1d3b684cc5c$export$9f8ac96af4b1b2ae (created by NativeBaseProvider)
    in ToastProvider (created by NativeBaseProvider)
    in PortalProvider (created by NativeBaseProvider)
    in HybridProvider (created by NativeBaseProvider)
    in ResponsiveQueryProvider (created by NativeBaseProvider)
    in RCTView (at View.windows.js:110)
    in View (created by CompatNativeSafeAreaProvider)
    in CompatNativeSafeAreaProvider (created by SafeAreaProvider)
    in SafeAreaProvider (created by NativeBaseProvider)
    in NativeBaseConfigProviderProvider (created by NativeBaseProvider)
    in NativeBaseProvider (at App.tsx:58)
    in AppModulesUpdateCheck (at App.tsx:54)
    in Initialization (at App.tsx:46)
    in ErrorBoundary (at App.tsx:45)
    in App (at renderApplication.js:50)
    in RCTView (at View.windows.js:110)
    in View (at AppContainer.js:92)
    in ChildrenWrapper (at wrapRootComponent.js:9)
    in _default (at wrapRootComponent.js:8)
    in Root (at AppContainer.js:107)
    in RCTView (at View.windows.js:110)
    in View (at AppContainer.js:119)
    in AppContainer (at renderApplication.js:43)
    in HP.HPX(RootComponent) (at renderApplication.js:60)

**Comment by Viraj-10** [comment]
Hi @bahri-dev, we currently don't support the windows app. We will consider it in future.

**Reply by bahri-dev** [reply]
Thanks for your confirming.
